packages/riipen-ui/src/components/Badge.jsx

```diff
@@ -172,6 +178,10 @@ class Badge extends React.Component {
             background-color: ${theme.palette.negative.main};
             color: ${theme.palette.negative.contrast};
           }
+          .muted {
+            background-color: ${theme.palette.grey[400]};
+            color: ${theme.palette.negative.contrast};
+          }
```

Comment:
I am going to assume these colours are confirmed somewhere as I dont know where to look 🙈

Reply:
`riipen-ui/src/themes/*.js` for all your colouring needs!

Reply:
I meant ui designs for the muted badge